test/old_tests/UnitTests/Composable.cpp

```diff
@@ -167,4 +167,20 @@ TEST_CASE("Composable conversions")
 {
     TestCalls(*make_self<Foo>());
     TestCalls(*make_self<Bar>());
-}
\ No newline at end of file
+}
+
+TEST_CASE("Composable get_interfaces")
+{
+    struct Foo : Composable::BaseT<Foo, IStringable> {
+        hstring ToString() const { return L"Foo"; }
+    };
+
+    auto obj = make<Foo>();
+    auto iids = winrt::get_interfaces(obj);
+    // BaseOverrides IID gets repeated twice. There are only 4 unique interfaces.
+    REQUIRE(iids.size() >= 4);
```

Comment:
Maybe its just me, but `>=4` seems overly conservative.  If there is a well-known repeated IID then can this assert the size is 5 instead?  Perhaps that's not ideal but it is at least testing the current behavior so that if it changes later the test will identify the behavior change.